temporalio/activity.py

```diff
@@ -135,6 +138,29 @@ def _logger_details(self) -> Mapping[str, Any]:
 _current_context: contextvars.ContextVar[_Context] = contextvars.ContextVar("activity")
 
 
+@dataclass
+class ActivityCancellationDetails:
```

Comment:
We may want to document that these cancellation details are set at the first time the activity is cancelled. They do not get updated if things change. So for instance, if an activity is cancelled due to worker shutdown, and then later the activity times out, this is not reflected here. Similarly, if an activity is paused and then unpaused, the paused doesn't get set back to false.

Reply:
added to the docstring